sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -478,6 +478,44 @@ def test_records_traverse_transform_with_mocks(self):
 
       assert_that(jobs, equal_to([job_reference]), label='CheckJobs')
 
+  @parameterized.expand([
+      param(compat_version=None),
+      param(compat_version="2.64.0"),
+  ])
+  def test_reshuffle_before_load(self, compat_version):
```

Comment:
I don't think this really tests what you want to test. You want:

- correctness of the transform with both settings of the flag
- if possible, some way to reproduce the issue you had before that is red before, green after

You also want to build an internal dataflow test of update compatibility with requested 2.64.0 version. I can show you a CL that does that if you haven't already seen them.